dashboard/test/controllers/api/v1/pd/workshops_controller_test.rb

```diff
@@ -29,6 +30,8 @@ class Api::V1::Pd::WorkshopsControllerTest < ::ActionController::TestCase
     )
 
     @standalone_workshop = create(:pd_workshop)
+
+    Pd::CourseFacilitator.create(facilitator: @csf_facilitator, course: Pd::Workshop::COURSE_CSF)
```

Comment:
nit, I'd prefer these 2 lines back to back, esp since it's called `@csf_facilitator`.

Or combined in one line, e.g.
```ruby
@csf_facilitator = create(:pd_course_facilitator, course: Pd::Workshop::COURSE_CSF).facilitator
```